foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/config/QueryHints.java

```diff
@@ -921,6 +921,15 @@ public class QueryHints {
      */
     public static final String RETURN_NAME_VALUE_PAIRS = "eclipselink.query-return-name-value-pairs";
 
+    /**
+     * "eclipselink.print-inner-join-in-where-clause"
+     * <p>Changes the way that inner joins are printed in generated SQL for the database.
+     * With a value of true, inner joins are printed in the WHERE clause, if false, inner joins are printed in the FROM clause.
+     * This query hint should override global/session switch {@link org.eclipse.persistence.internal.databaseaccess.DatabasePlatform#setPrintInnerJoinInWhereClause(boolean)}
+     * @see org.eclipse.persistence.internal.databaseaccess.DatabasePlatform#setPrintInnerJoinInWhereClause(boolean)
+     */
+    public static final String PRINT_INNER_JOIN_IN_WHERE_CLAUSE = "eclipselink.print-inner-join-in-where-clause";
```

Comment:
Is there a way to make the constant name/value shorter, ie avoid the `print` word?